src/contexts/FileBrowserContext.tsx

```diff
@@ -120,7 +120,7 @@ export const FileBrowserContextProvider = ({
     const url = getFileFetchPath(path);
 
     try {
-      const response = await sendGetRequest(url, cookies['_xsrf']);
+      const response = await sendFetchRequest(url, 'GET', cookies['_xsrf']);
```

Comment:
We should add some unit testing. Not sure how that slips through the cracks

Reply:
Something went wrong with a merge, I think. Finding out what happened using `git bisect` is an exercise left to the reader.